### PR TITLE
[genians-1724] Product name and source modification

### PR DESCRIPTION
**Pull request by genians-rnd**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [O] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Genian NAC product name and source modification (Genians-> Genian NAC)

## Screenshots
N/A

## Minimum version of Demisto
- [ ] 5.0.0
- [O] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by adi88d** [comment]
@genians-rnd please fill the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**Comment by genians-rnd** [comment]
@adi88d I have finished writing.

File: Packs/GenianNAC/Integrations/GenianNAC/GenianNAC.py [hunk 1]
```diff
@@ -187,9 +187,9 @@ def main():
         if demisto.command() == "test-module":
             get_ip_nodeid('8.8.8.8')
             demisto.results('ok')
-        elif demisto.command() == 'genians-assign-ip-tag':
```
Comment (adi88d) [inline Packs/GenianNAC/Integrations/GenianNAC/GenianNAC.py:190]: Note that when you update commands name and outputs is breaking backward compatibility.

File: Packs/GenianNAC/Integrations/GenianNAC/GenianNAC.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 commonfields:
-  id: Genians
+  id: Genian NAC
```
Comment (adi88d) [inline Packs/GenianNAC/Integrations/GenianNAC/GenianNAC.yml:2]: please revert the id filed, you can change the display and the name only

**Review by adi88d — COMMENTED** [review]
- please revert the pack folder and the integration folder name to `Genians`
- update the docker image tag to the latest version -  3.8.6.12176
- update release notes - run `demisto-sdk update-release-notes -i Packs/GenianNAC

File: Packs/GenianNAC/Integrations/GenianNAC/GenianNAC.py (hunk 1 above)
Comment (genians-rnd) [reply Packs/GenianNAC/Integrations/GenianNAC/GenianNAC.py:190]: I will revert it back.

File: Packs/GenianNAC/Integrations/GenianNAC/GenianNAC.yml (hunk 2 above)
Comment (genians-rnd) [reply Packs/GenianNAC/Integrations/GenianNAC/GenianNAC.yml:2]: I will revert it back.

**Comment by genians-rnd** [comment]
Thank you.
I will request PR again.

**Pull request closed without merging**